src/index.html

```diff
@@ -1,13 +1,429 @@
+                </h3>
+
+                <div class="testimonial__role text__gray">
+                  Chief Design Officer
+                </div>
+
+                <p class="testimonial__about">
+                  When my lovely valley of steam rises and half-day sun is above an impermeable
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title text__title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title text__gray">
+                  Phone
+                </div>
+
+                <a
+                  href="tel:+1 234 5555-55-55 "
+                  class="contact-us__info-link"
+                >
+                  +1 234 5555-55-55
+                </a>
+              </div>
+
+              <div class="contact-us__info">
+                <div class="contact-us__info-title text__gray">
+                  Email
+                </div>
+
+                <a
+                  href="mailto: hello@miami.com"
+                  class="contact-us__info-link"
+                >
+                  hello@miami.com
+                </a>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--4-5">
+              <div class="contact-us__info contact-us__info--last">
+                <div class="contact-us__info-title text__gray">
+                  Address
+                </div>
+
+                <a
+                  href="https://goo.gl/maps/nM8MDc9b5HshyxqKA"
+                  class="contact-us__info-link"
+                >
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
+                </a>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--8-12">
+              <form action="">
+                <input
+                  type="text"
+                  name="name"
+                  placeholder="Name"
+                  class="contact-us__field form-field"
+                  required
+                >
+
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="Email"
+                  class="contact-us__field form-field"
+                  required
+                >
+
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="
+                    contact-us__field
+                    contact-us__field--last
+                    form-field
+                    form-field--textarea"
+                  required>
+                </textarea>
```

Comment:
```suggestion
                  required
               ></textarea>
```
зараз між закриваючим і відкриваючим тегами є пробіли забери їх